samples/snippets/speech_to_storage_beta.py

```diff
@@ -40,20 +49,39 @@ def export_transcript_to_storage_beta(
         audio=audio, config=config, output_config=output_config
     )
 
-    # Create the speech client
+    # create the speech client
     speech_client = speech.SpeechClient()
 
+    # create the storage client
+    storage_client = storage.Client()
+
+    # run the recognizer to export transcript
     operation = speech_client.long_running_recognize(request=request)
 
     print("Waiting for operation to complete...")
-    response = operation.result(timeout=90)
+    operation.result(timeout=90)
+
+    # get bucket with name
+    bucket = storage_client.get_bucket(bucket_name)
+
+    # get blob from bucket
+    blob = bucket.get_blob(object_name)
+
+    # get content as string
+    results_string = blob.download_as_string()
+
+    # get transcript exported in storage bucket
+    storage_transcript = types.RecognizeResponse.from_json(
+        results_string, ignore_unknown_fields=True
```

Comment:
ignore_unknown_fields should not be necessary.

Reply:
I am also constrained to use the ```ignore_unknown_fields``` as there could be issues parsing from the string json.

Reply:
I guess makes sense, since new fields can be added before the api libraries are updated.